The first program will print `3`. The second program will throw error because is defined in inner scope of do/end block.
